<?xml version="1.0"?>
<launch>
        <node pkg="barrel_detector" name="barrel_detector" type="barrel_detector" output="screen">
                <param name="debuging_" type="bool" value="false" />
                <param name="probeblisitic_" type="bool" value="true" />
                <param name="min_radius_" type="double" value="20" />
                <param name="max_radius_" type="double" value="40" />
                <param name="min_circle_percentage_" type="double" value="0.50" />
          </node>
</launch>
